packages/front-end/components/Features/FeatureImplementationModal.tsx

```diff
@@ -53,7 +54,7 @@ if (myFeature) { ...
 console.log(growthbook.getFeatureValue(${JSON.stringify(
         feature.id
       )}), "fallback value");`,
-      docs: "https://docs.growthbook.io/lib/react",
+      docs: getDocsLink("tsx"),
```

Comment:
I would change the type for this to expect the id of a doc section instead of a full URL path.  That will reduce the chance for typos and make the code simpler (we'll be able to just use the `<DocLink>` component instead of building our own <a> tag from scratch.